packages/react-web/src/inbox/helpers/__tests__/conversationIdUtils.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  buildConversationId,
+  extractPeerProfileId,
+  parseConversationId,
+} from '../conversationIdUtils';
+
+describe('Given a collection of inbox helper functions', () => {
+  describe('when calling buildConversationId', () => {
```

Comment:
```suggestion
  describe(`when calling ${buildConversationId.name}`, () => {
```

makes future renaming refactoring less error prone.

Other tests in this file might need this change.